assets/phosphor-icons.css

```
@@ -0,0 +1,70 @@
+/* Phosphor Icons 2.1.2, regular weight — https://phosphoricons.com */
+@font-face {
+  font-family: "Phosphor";
```

Stylelint (17.14.1)

[error] Expected no quotes around "Phosphor" (font-family-name-quotes)

(font-family-name-quotes)